part2/phonebook/src/App.js

```diff
@@ -1,40 +1,108 @@
 import React, { useState, useEffect } from 'react'
-import axios from 'axios'
+import personsService from './services/persons'
+import './index.css'
 import Filter from './components/Filter'
 import PersonForm from './components/PersonForm'
 import Persons from './components/Persons'
+import Notification from './components/Notification'
+import Footer from './components/Footer'
 
 const App = () => {
-  const [persons, setPersons] = useState([]) 
+  const [ persons, setPersons ] = useState([]) 
   const [ newName, setNewName ] = useState('')
   const [ newNumber, setNewNumber ] = useState('')
   const [ newFilter, setNewFilter ] = useState('')
+  const [ notificationMessage, setNotificationMessage] = useState(null)
+  const [ type, setType ] = useState('note')
+  //TODO tu by som mal taku otazku na niekoho skusenejsieho ci sa to da 
+  //robit aj inaksie ako stale vytvarat useState. Popripade ak je to ok, 
+  //tak ci to nie je problem mat vela useState.
```

Comment:
Tu by som mal taku otazku na niekoho skusenejsieho ci sa to da robit aj inaksie ako stale vytvarat useState. Popripade ak je to ok, tak ci to nie je problem mat vela useState.